Sprint-1/errors/4.js

```diff
@@ -1,2 +1,4 @@
-const 12HourClockTime = "20:53";
-const 24hourClockTime = "08:53";
\ No newline at end of file
+const timeIn12HourFormat  = "20:53";
```

Comment:
Did you notice the variable names do not quite match the values assigned to the variables?